app/services/exports/trainee_search_data.rb

```diff
@@ -130,6 +130,8 @@ def funding_method(trainee)
         FUNDING_TYPE_ENUMS[:grant]
       elsif [trainee.applying_for_bursary, trainee.applying_for_scholarship, trainee.applying_for_grant].include?(false)
         "not funded"
+      elsif !FundingManager.new(trainee).funding_available?
```

Comment:
We should make method for returning the `FundingManager` instance as this is duplicated on 139.